esphome/components/at581x/binary_sensor.py

```diff
@@ -0,0 +1,28 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import binary_sensor
+from esphome.const import (
+    DEVICE_CLASS_MOTION,
+    ICON_MOTION_SENSOR,
+)
+
+from . import CONF_AT581X_ID, AT581XComponent
+
+DEPENDENCIES = ["at581x"]
+
+CONF_MOTION_DETECTED = "motion_detected"
+
+CONFIG_SCHEMA = {
+    cv.GenerateID(CONF_AT581X_ID): cv.use_id(AT581XComponent),
+    cv.Optional(CONF_MOTION_DETECTED): binary_sensor.binary_sensor_schema(
```

Comment:
Can we please change this to be in line with the other motion/radar/mmwave sensors?
```suggestion
    cv.Optional(CONF_HAS_TARGET): binary_sensor.binary_sensor_schema(
```

Reply:
The alternative is to not make this a child sensor of the config at all, but just directly on the CONFIG_SCHEMA since it is the only one.

Reply:
I'm not 100% sure I get it here. This radar detects motion not static target (unlike 24GHz or 60GHz? IIUC). If you stop moving in front of the radar, the detection signal will go down. 

In the datasheet, there's also a light detection sensor, but I don't know if it's an error in the Chinese to English translation (in that case, light would mean RF emission, I'm not sure), or if it's actually here. 

What's sure is that on Box S3, the radar is under a opaque plastic cover so it can't detect light.

What do you mean about not making a child sensor, do you have an example I can look at?